packages/back-end/src/models/FeatureModel.ts

```diff
@@ -153,6 +157,50 @@ async function logFeatureUpdatedEvent(
   return emittedEvent.id;
 }
 
+/**
+ * @param organization
+ * @param feature
+ * @returns event.id
+ */
+async function logFeatureCreatedEvent(
+  organization: OrganizationInterface,
+  feature: FeatureInterface
+): Promise<string> {
+  const payload: FeatureCreatedNotificationEvent = {
+    object: "feature",
+    event: "feature.created",
+    data: feature,
```

Comment:
wondering if we should nest `feature` in its own property (`data.current`) rather than directly on `data`, similar to feature.deleted.

Reply:
Yeah, I like the idea of using `data.current` to match the other event payloads

Reply:
I don't personally think that's necessary, and this type was defined a while back and approved in a previous PR.

The reason we have `current` and `previous` for updated events is because we want to share both states.

The reason I used `previous` for deleted events is to make it clear that is the previous state and that the feature doesn't exist anymore.

In the case of created events, using `data.current` seems redundant to me, but I'm ok with adding that if we'd like to remain consistent. 

Maybe @jdorn wants to weigh in?

Reply:
Looks like you sent your reply before I sent mine. Sure, I can update it to use `data.current`.